### PR TITLE
Team2 david

### PR DESCRIPTION
**Pull request by gormley01**
## Description

Added the acceptable bullets under the participation section.

### Type of Change

- [ ] Feature
- [ ] Bug fix
- [x] Documentation

## Contributors

- @gormley01 

## Reminder

All GitHub Actions checks must be in a passing state in this branch or fork before this pull request can be merged.

### CODE REVIEW

**Review by brendengray — APPROVED** [review]
Looks good to me sir.

**Review by jjumadinova — CHANGES_REQUESTED** [review]
The content looks good, however there seem to be conflicts that need to be resolved. Please resolve conflicts and request a review again for approval.

File: conduct.md [hunk 1]
```diff
@@ -56,9 +56,11 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+Students are encouraged to:
+
+- Participate in class discussions.
```
Comment (solisa986) [inline conduct.md]: Please provide an additional sentence for each bullet point that provides an example/definition of 'participation', 'when needed', and 'regularly'. For example, is participation limited to a certain number of student input per class? Is 'when needed' up to the student to decide or for the team? Does attending regularly rely on a certain number of absences and does it apply to the lab and practical sessions?

**Review by solisa986 — CHANGES_REQUESTED** [review]
Good job on your first pull request! I have outlined some changes that need to be made prior to requesting another review from me. Once you have made the changes outlined in my comment, please make sure to resolve the conversation and then re-request my review.

**Review by Danniyb — APPROVED** [review]

**Review by solisa986 — APPROVED** [review]
looks good!

**Pull request closed without merging**